examples/guestbook/README.md

```diff
@@ -464,7 +464,7 @@ When you go to localhost:8000, you might not see the page at all.  Testing it wi
    ==> default: curl: (56) Recv failure: Connection reset by peer
 ```
 
-This means the web frontend isn't up yet.  Wait a while, possibly about 2 minutes or more, depending on your set up. Also, run a *watch* on docker ps, to see if containers are cycling on and off or not starting.
+This means the web frontend isn't up yet.  So "reset by peer", is occuring because you are trying to access the right port, but nothing is bound to that port yet.  Wait a while, possibly about 2 minutes or more, depending on your set up. Also, run a *watch* on docker ps, to see if containers are cycling on and off or not starting.
```

Comment:
Can you reword this slightly. I'd suggest:

This means the web frontend isn't up yet ("reset by peer" is occurring because you are trying to access the right port, but nothing is bound to that port yet).